app/upgrades.go

```diff
@@ -9,57 +12,26 @@ import (
 // NOTE: When performing upgrades, make sure to keep / register the handlers
 // for both the current (n) and the previous (n-1) upgrade name. There is a bug
 // in a missing value in a log statement for which the fix is not released
-const IgniteCLIRemovalUpgradeHandler = "ignite-cli-removal-upgrade"
-
-// 1.0.2beta upgrades
-const Upgrade102 = "1.0.2beta"
-const Upgrade102CommitTimeout = "1.0.2beta-commit-timeout"
-
-// 1.0.3beta
-const Upgrade103 = "1.0.3beta"
-
-// 1.0.4beta
-// this will introduce the oracle module as well
-const Upgrade104 = "1.0.4beta"
-
-// 1.0.5beta
-const Upgrade105 = "1.0.5beta upgrade"
-
-// 1.0.6beta
-const Upgrade106 = "1.0.6beta"
-
```

Comment:
Should be fine to remove as shown in the test that we can remove historical handlers